### ISSUE TITLE
No package versions were returned. when using local link deps

### ISSUE THREAD

**Issue by Wyzix33** [raineorshine/npm-check-updates#1165]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 18.6`

---

## Steps to Reproduce

Dependencies:

```json
{
  "name": "form",
  "version": "1.0.0",
  "main": "form.js",
  "author": "Gabriel",
  "license": "No License",
  "private": true,
  "dependencies": {
    "editor": "link:../editor",
    "event": "link:../event",
    "select": "link:../select",
    "socket": "link:../socket",
    "upload": "link:../upload",
    "utils": "link:../utils"
  }
}
```

Steps:

## Current Behavior
Hi, just looked at my libs, the ones triggering that warning 'No package versions were returned... ' are libs that have only local link deps.
running ncu -u.
But if a lib has at least a normal dependency it doesn't trigger the warning, for example this one is fine
```
{
 "name": "table",
 "version": "1.0.0",
 "description": "tables",
 "main": "table.js",
 "author": "Gabriel",
 "license": "No License",
 "private": true,
 "dependencies": {
  "event": "link:../event",
  "split.js": "^1.6.5",
  "utils": "link:../utils"
 }
}
``` 
## Expected Behavior
No warning...

Thanks

**Comment by raineorshine** [comment]
Hi, thanks for reporting. 

Can you point to any documentation on local link deps? I am not familiar with them.

**Comment by Wyzix33** [comment]
Hi, 
[yarn link](https://classic.yarnpkg.com/en/docs/cli/link)
[npm install](https://docs.npmjs.com/cli/v8/commands/npm-install) see npm install folder

it's the same as installing a lib from a local folder
`npm install c:\test\utils\`
this will result in 
```
  "dependencies": {
        "utils": "file:utils"
  }
```
but i'm using yarn and running `yarn add link:c:\test\utils`
gives me 
```
  "dependencies": {
    "utils": "link:c:\\test\\utils"
  }
```
it's the same thing and both trigger that `No package versions were returned....` 
So as long as those libs are local they can be ignored without checking if a new version is available.
Thanks

**Comment by raineorshine** [comment]
Thanks, that was helpful!

I have published a fix to `v15.3.4`.

**Comment by Wyzix33** [comment]
Nice work!
Thanks
